### PR TITLE
KOGITO-3930 Align archetype with settings from Quarkus

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Review by evacchi — APPROVED** [review]

File: archetypes/kogito-quarkus-archetype/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -77,6 +77,12 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <version>${surefire.version}</version>
+        <configuration>
+          <systemPropertyVariables>
+            <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
```
Comment (radtriste) [inline archetypes/kogito-quarkus-archetype/src/main/resources/archetype-resources/pom.xml:82]: could we make that property `org.jboss.logmanager.LogManager` more generic in build pom or kogito-runtimes pom ?

**Review by radtriste — APPROVED** [review]
just minor comment

File: archetypes/kogito-quarkus-archetype/src/main/resources/archetype-resources/pom.xml (hunk 1 above)
Comment (cristianonicolai) [reply archetypes/kogito-quarkus-archetype/src/main/resources/archetype-resources/pom.xml:82]: @radtriste not sure I got the idea... you mean to be reused across our kogito-runtimes poms? like in the pluginManagement section?

Comment (radtriste) [reply archetypes/kogito-quarkus-archetype/src/main/resources/archetype-resources/pom.xml:82]: that was just an idea to have

`<quarkus.logging.manager>org.jboss.logmanager.LogManager</quarkus.logging.manager>` in kogito-build-parent

and use it here 
```
<systemPropertyVariables>
            <java.util.logging.manager>${quarkus.logging.manager}</java.util.logging.manager>
```
and in other places as well

I think it is a nice to have ;)

Comment (cristianonicolai) [reply archetypes/kogito-quarkus-archetype/src/main/resources/archetype-resources/pom.xml]: @radtriste thanks, added to the pr :)

File: archetypes/kogito-quarkus-archetype/src/main/resources/archetype-resources/pom.xml [hunk 2]
```diff
@@ -96,25 +104,31 @@
...
               <execution>
                 <goals>
-                  <goal>native-image</goal>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
                 </goals>
+                <configuration>
+                  <systemPropertyVariables>
+                    <native.image.path>${project.build.directory}/${project.build.finalName}-runner</native.image.path>
+                    <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
```
Comment (radtriste) [inline archetypes/kogito-quarkus-archetype/src/main/resources/archetype-resources/pom.xml:125]:
```suggestion
                    <java.util.logging.manager>${quarkus.logging.manager}</java.util.logging.manager>
```

**Review by radtriste — CHANGES_REQUESTED** [review]

**Comment by radtriste** [comment]
maybe change also in other pom where it configures `org.jboss.logmanager.LogManager` ?

**Comment by cristianonicolai** [comment]
@radtriste missed that, fixed.

**Review by radtriste — APPROVED** [review]

**Pull request merged**